dojo/settings/settings.dist.py

```diff
@@ -177,6 +177,11 @@
         "Lastname": "last_name",
     }),
     DD_SAML2_ALLOW_UNKNOWN_ATTRIBUTE=(bool, False),
+    # similar to DD_SOCIAL_AUTH_AZUREAD_TENANT_OAUTH2_GROUPS_FILTER, regular expression for which SAML2 groups to map to Dojo groups (with same name)
+    # Groups need to already exist in Dojo. And if value is not set, no group processing is done
+    DD_SAML2_GROUPS_FILTER=(str, ""),
```

Comment:
Can you call it `_READERS` as it adds a Reader role? In the future 
```suggestion
    DD_SAML2_GROUPS_FILTER_READERS=(str, ""),
```
Or even more future-proof (if new roles would be created): dict with mapping to roles
```suggestion
    DD_SAML2_GROUPS_FILTER=(dict, {"readers": ""}),
```
More info, about how dict works with `environ`
- https://django-environ.readthedocs.io/en/latest/types.html#environ-env-dict
- https://django-environ.readthedocs.io/en/latest/tips.html#complex-dict-format

Reply:
At first sight, I agree this would make sense, but does it really? When would anyone not be just reader in a group where the membership is expected to be managed by IdP?

Noone should be able to add/remove people from the group in Dojo, they should do it in IdP, otherwise that person will eventually be removed from the group (upon login)

Maybe I'm missing some use case in my mind, please let me know if so. Otherwise this would just increase complexity of the configuration without any added benefit